src/ERPNext/Client/Filters.hs

```diff
@@ -11,20 +11,23 @@ import Data.Text (Text, intercalate)
 import Data.Time.Calendar (Day)
 import ERPNext.Client.Helper (urlEncode, quote, tshow)
 
--- TODO: refactor this? rename to filter and parameterize each term with fieldname and value?
-data FilterOperator
-  = Eq
-  | NotEq
-  | Greater
-  | GreaterOrEq
-  | Less
-  | LessOrEq
-  | Like
-  | NotLike
-  | In
-  | NotIn
-  | Between
-  | Is
+newtype Fieldname = Fieldname Text
```

Comment:
Oh, I think I confused this. I wanted to suggest a type alias, so it should be just `type Fieldname = Text`